capstone-rs/src/capstone.rs

```diff
@@ -365,6 +365,44 @@ impl Capstone {
         }
     }
 
+    /// Get the registers are which are read to and written to, in that order.
+    pub fn regs_access(&self, insn: &Insn) -> Option<CsResult<(Vec<RegId>, Vec<RegId>)>> {
```

Comment:
Instead of using a tuple, please make a new struct that makes it clear what is the read and write fields:

```rust
pub struct RegAccess {
    read: Vec<RegId>,
    write: Vec<RegId>,
}
```

The return type would be `CsResult<RegAccess>`.

Reply:
I used that struct but made the fields public, is it good now?